<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detector</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="menuFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>338</width>
        <height>514</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">width:20%;
height:100%;
background-color: rgb(250, 247, 240);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonHome">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:17pt; font-weight:792; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">height:80%;
width:100%;
margin-top:30%;
background-color:rgb(221,184,146);
font-size:16pt;
font: &quot;黑体&quot;;
font-weight:900;
color:rgb(51, 57, 85);</string>
         </property>
         <property name="text">
          <string>首页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonBackground">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">height:80%;
width:90%;
margin-top:10%;

background-color:rgb(221,184,146);
font-size:15pt;
font: &quot;黑体&quot;;
font-weight:900;
color:rgb(51, 57, 85);</string>
         </property>
         <property name="text">
          <string>背景检测</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonObject">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">height:80%;
width:100%;
margin-top:10%;

background-color:rgb(221,184,146);
font-size:15pt;
font: &quot;黑体&quot;;
font-weight:900;
color:rgb(51, 57, 85);</string>
         </property>
         <property name="text">
          <string>目标跟踪</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonExit">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">width:100%;
height:20%;
color: rgb(255, 142, 62);
background-color:rgb(221,184,146);
font-size:10pt;
font: &quot;黑体&quot;;
font-weight:900;</string>
         </property>
         <property name="text">
          <string>⚪ 退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(250, 247, 240);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="pageHome">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="labelHome">
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;黑体&quot;;
color: rgb(89, 69, 69);
margin-top:10%</string>
          </property>
          <property name="text">
           <string>首页</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowserIntroduction">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(215, 191, 157);
color:black;
margin-top:20%;
border-radius:15;</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#010000;&quot;&gt;本系统集成了一个背景检测系统以及目标跟踪系统。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true"/>
        </item>
        <item>
         <widget class="QLabel" name="labelTail">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;黑体&quot;;
color: rgb(89, 69, 69);
margin-top:10%;
text-align:right;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;本程序xxx请见&lt;a href=&quot;https://github.com/CharRic/Detector/tree/main&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;网页&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageBackground">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="labelBackground">
          <property name="styleSheet">
           <string notr="true">font-size:20pt;
margin-left:20%;
margin-top:10%;
margin-bottom:10%;
font-weight:bold;</string>
          </property>
          <property name="text">
           <string>背景检测</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetBackground" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QWidget" name="widgetSelection" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="labelBackgroundSelection">
                <property name="styleSheet">
                 <string notr="true">font-size:14pt;</string>
                </property>
                <property name="text">
                 <string>请选择视频文件:</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonBackgroundSelection">
                <property name="styleSheet">
                 <string notr="true">font: 11pt &quot;黑体&quot;;
background-color:rgb(221,184,146);</string>
                </property>
                <property name="text">
                 <string> 选择文件 </string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(215, 192, 174);
border:2px solid black;
margin-right:100%;

margin-left:100%;</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="labelBackgroundImg">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">heiht:100%;
border:0;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetBackgroundTail" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QTabWidget" name="tabWidgetBackground">
             <property name="toolTipDuration">
              <number>-1</number>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: white;
font: 13pt &quot;楷体&quot;;</string>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="tabBackgroundInstruction">
              <attribute name="title">
               <string>教程</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QTextBrowser" name="textBrowserBackgroundInstruction"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tabBackgroundInformation">
              <attribute name="title">
               <string>信息台</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QTextBrowser" name="textBrowserBackgroundInformation"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonBackgroundDetection">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">height:100%;
width:100%;
margin-left:20%;
margin-right:20%;
border-image:url(&quot;./icon/right-circle.png&quot;);
background-color:rgba(255, 255, 255, 0);
border-radius:50%;
margin-top:20%;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageObject">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="labelObject">
          <property name="styleSheet">
           <string notr="true">font-size:20pt;
margin-left:20%;
margin-top:10%;
margin-bottom:10%;
font-weight:bold;</string>
          </property>
          <property name="text">
           <string>目标跟踪</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetObject" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QWidget" name="widgetObjectSelection" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="labelObjectSelection">
                <property name="styleSheet">
                 <string notr="true">font-size:14pt;</string>
                </property>
                <property name="text">
                 <string>请选择视频文件:</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonObjectSelection">
                <property name="styleSheet">
                 <string notr="true">font: 11pt &quot;黑体&quot;;
background-color:rgb(221,184,146);</string>
                </property>
                <property name="text">
                 <string> 选择文件 </string>
                </property>
               </widget>
              </item>
             </layout>
             <zorder>labelObjectSelection</zorder>
             <zorder>pushButtonObjectSelection</zorder>
             <zorder>horizontalSpacer</zorder>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widgetObjectVideo" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QWidget" name="widget_2" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(215, 192, 174);
border:2px solid black;
margin-right:50%;
margin-left:50%;</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="labelObjectImg">
                   <property name="styleSheet">
                    <string notr="true">border:0;</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widgetObjectChoice" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">border:1px solid black;
font: 9pt &quot;新宋体&quot;;</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QRadioButton" name="radioButtonObjectStatic">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border:none;</string>
                   </property>
                   <property name="text">
                    <string>静止摄像机</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButtonObjectMove">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">border:none;</string>
                   </property>
                   <property name="text">
                    <string>运动摄像机</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetObjectTail" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QTabWidget" name="tabWidgetObject">
             <property name="toolTipDuration">
              <number>-1</number>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: white;
font: 13pt &quot;楷体&quot;;</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tabObjectInstruction">
              <attribute name="title">
               <string>教程</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QTextBrowser" name="textBrowserObjectInstruction">
                 <property name="styleSheet">
                  <string notr="true">font: 10pt &quot;Times New Roman&quot;;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tabObjectInformation">
              <attribute name="title">
               <string>信息台</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QTextBrowser" name="textBrowserObjectInformation"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonObjectDetection">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">height:100%;
width:100%;
margin-left:20%;
margin-right:20%;
border-image:url(&quot;./icon/right-circle.png&quot;);
background-color:rgba(255, 255, 255, 0);
border-radius:50%;
margin-top:20%;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>10</width>
               <height>10</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuList">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>  帮助  </string>
    </property>
    <addaction name="actionAlgorithm"/>
    <addaction name="separator"/>
    <addaction name="actionInformation"/>
   </widget>
   <addaction name="menuList"/>
  </widget>
  <action name="actionAlgorithm">
   <property name="icon">
    <iconset>
     <normaloff>../icon/wendang.png</normaloff>../icon/wendang.png</iconset>
   </property>
   <property name="text">
    <string>算法原理</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="icon">
    <iconset>
     <normaloff>../icon/banquan.png</normaloff>../icon/banquan.png</iconset>
   </property>
   <property name="text">
    <string>关于Detector</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
